apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -56,82 +56,25 @@ export default function CurriculumQuickAssign({updateSection, sectionCourse}) {
 
   return (
     <div>
-      <h3>{i18n.assignACurriculum()}</h3>
-      <h5>{i18n.useDropdownMessage()}</h5>
-      <div className={moduleStyles.buttonRow}>
-        <div className={moduleStyles.buttonsInRow}>
-          <Button
-            id={'uitest-elementary-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseBlocksGradeBandsElementary()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.ELEMENTARY
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.ELEMENTARY);
-            }}
-          />
-          <Button
-            id={'uitest-middle-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseBlocksGradeBandsMiddle()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.MIDDLE
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.MIDDLE);
-            }}
-          />
-          <Button
-            id={'uitest-high-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseBlocksGradeBandsHigh()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.HIGH
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.HIGH);
-            }}
-          />
-          <Button
-            id={'uitest-hoc-button'}
-            className={moduleStyles.buttonStyle}
-            text={i18n.courseOfferingHOC()}
-            size={Button.ButtonSize.large}
-            icon={
-              marketingAudience === MARKETING_AUDIENCE.HOC
-                ? 'caret-up'
-                : 'caret-down'
-            }
-            onClick={e => {
-              e.preventDefault();
-              updateMarketingAudience(MARKETING_AUDIENCE.HOC);
-            }}
-          />
-          <input
-            checked={decideLater}
-            className={moduleStyles.input}
-            type="checkbox"
-            id="decide-later"
-            onChange={toggleDecideLater}
-          />
-          <label className={moduleStyles.decideLater} htmlFor="decide-later">
-            {i18n.decideLater()}
-          </label>
-        </div>
+      <div className={moduleStyles.input}>
```

Comment:
adding a second level of styles allows us to override the usual input scss styling